src/domain/entities/generic/Collection.ts

```diff
@@ -15,6 +15,9 @@ import { HashMap } from "./HashMap";
  * ```
  */
 
+// HMIS Tally Sheets not implemented methods from lodash:
+// "lodash": "4.17.21", --> _.isEmpty and _.intersection to be added to Collection
```

Comment:
Why the comment? is intersection needed?

Reply:
It was before and I added the comment for my future self. Now that the feature is almost done, the use of intersection was removed in the pathway of moving the logic that it was in the usecase, into the entity itself